.circleci/config.yml

```diff
@@ -5,20 +5,17 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v3-pkg-cache
+            - go-mod-v4-{{ checksum "go.sum" }}
             - v5-node-cache
       - run:
-          name: Install glide
-          command: sudo apt-get update -y && sudo apt-get install -y golang-glide
-      - run:
-          name: Install dependencies using glide
-          command: glide install
-      - run:
-          name: Fetch astilectron-bundler
-          command: go get -u github.com/asticode/go-astilectron-bundler/...
+          name: Download Go Modules
+          command: |
+            export GOPROXY=https://goproxy.io,https://proxy.golang.org,https://goproxy.cn
```

Comment:
We should test only with proxy.Golang.org

Reply:
I tried that yesterday and it failed so had to add the others, I think its actually this https://goproxy.cn one that has that lib but kept the others for completeness.

Reply:
Ok then let’s include all 3 in the readme too so it is consistent with the CI setup